audioapp/templates/audioapp/base.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html>
+<head>
+</head>
+<body>
+    <header>
+        <nav>
+            {% if user.is_authenticated %}
+            <form method="post" action="{% url 'account_logout' %}">
+                {% csrf_token %}
+                <button type="submit">Sign Out</button>
+            </form>
```

Comment:
Still some indentation issues here. Since form is inside the if block, it should be indented.